cache-config/testing/docker/trafficserver/Dockerfile

```diff
@@ -17,54 +17,106 @@
 
 ###############################################################
 # Dockerfile to build Traffic Server RPM
-# Based on CentOS 7 for ATS RPM ot match the ort_test 
-# container running CentOS 7.
+# Defaults to CentOS 8, see RHEL_VERSION 
 ###############################################################
 
-ARG RHEL_VERSION=7
+# This sets the minimum RHEL_VERSION to 8
+# You may override and use RHEL_VERSION 7 using:
+#   'docker-compose build --build-args RHEL_VERSION=7 ats_build'
+# and followed by
+#   'docker-compose run -e RHEL_VERSION=7 ats_build'
+#
+# For the GHA workflow, override this in .github/workflows/cache-config-tests.yml
+ARG RHEL_VERSION=${RHEL_VERSION:-8}
 FROM centos:${RHEL_VERSION}
-ARG RHEL_VERSION=7
+ARG RHEL_VERSION=${RHEL_VERSION:-8}
```

Comment:
Same here. The `$RHEL_VERSION` build arg is always undefined before this line. Because `${RHEL_VERSION:-8}` always defaults to `8`, it can be just `RHEL_VERSION=8`.

Reply:
fixed